contracts/src/v0.1/RequestResponseCoordinator.sol

```diff
@@ -615,25 +630,39 @@ contract RequestResponseCoordinator is
             return totalFee;
         } else {
             // [regular] account
-            uint64 reqCount = sPrepayment.getReqCount(rc.accId);
-            uint256 serviceFee = calculateServiceFee(reqCount) * rc.numSubmission;
-            uint256 operatorFee = sPrepayment.chargeFee(rc.accId, serviceFee);
-            uint256 feePerOperator = operatorFee / oraclesLength;
-
-            uint256 paid;
-            for (uint256 i = 0; i < oraclesLength - 1; ++i) {
-                sPrepayment.chargeOperatorFee(rc.accId, feePerOperator, oracles[i]);
-                paid += feePerOperator;
-            }
+            (, uint64 reqCount, , , uint8 accType) = sPrepayment.getAccount(rc.accId);
+            if (accType == 1 || accType == 2) {
+                //decrease period request number
+                sPrepayment.decreasePeriodReq(rc.accId);
```

Comment:
Because the payment is related to service fee proposed above, I will try to finish my thought here. This portion of code should also be move to `CoordinatorBase`.

```Solidity
(, uint64 reqCount, , , uint8 accType) = sPrepayment.getAccount(rc.accId);
if (accType == AccountType.FIAT_SUBSCRIPTION) {
    sPrepayment.increaseReqCount(rc.accId);
    return 0;
} else {
    if (accType == AccountType.KLAY_SUBSCRIPTION) {
        sPrepayment.increaseReqCount(rc.accId);
    }
    uint256 serviceFee;

    if (accType == AccountType.KLAY_SUBSCRIPTION) {
        uint256 index = (now - sStartTime) / sPeriod;
        if (!sSubscriptionPaid[index]) {
            serviceFee = sSubscriptionPrice;
            sSubscriptionPaid[index] = true;
         } else {
             return 0;
         }
    } else if (accType == AccountType.KLAY_DISCOUNT) {
        serviceFee = (serviceFee * sPrepayment.getFeeRatio(rc.accId)) / 10_000;
    } else if (accType == AccountType.KLAY_REGULAR) {
        serviceFee = calculateServiceFee(reqCount) * rc.numSubmission;
    }

    ...
}
```